scootapi/client/targets.go

```diff
@@ -0,0 +1,7 @@
+package client
+
+// the path to any pants tests that should pass go here,
+// scootapi test_targets will test all of them.
```

Comment:
I'd add a TODO(rcouto): this is a big hack; move this into a file and read it (also, move this into a separate script)